PerformanceCalculatorGUI/Screens/ObjectInspection/OsuObjectInspectorDrawable.cs

```diff
@@ -58,8 +58,8 @@ private void refresh()
             {
                 panel.AddParagraph($"Strain Time: {entry.DifficultyHitObject.StrainTime:N3}");
                 panel.AddParagraph($"Aim Difficulty: {AimEvaluator.EvaluateDifficultyOf(entry.DifficultyHitObject, true):N3}");
-                panel.AddParagraph($"Speed Difficulty: {SpeedEvaluator.EvaluateDifficultyOf(entry.DifficultyHitObject, 20):N3}"); // OD10, not possible to get difficulty stats from here?
-                panel.AddParagraph($"Rhythm Difficulty: {RhythmEvaluator.EvaluateDifficultyOf(entry.DifficultyHitObject, 20):N3}");
+                panel.AddParagraph($"Speed Difficulty: {SpeedEvaluator.EvaluateDifficultyOf(entry.DifficultyHitObject):N3}"); // OD10, not possible to get difficulty stats from here?
```

Comment:
comment is no longer valid
```suggestion
                panel.AddParagraph($"Speed Difficulty: {SpeedEvaluator.EvaluateDifficultyOf(entry.DifficultyHitObject):N3}");
```